eofConvention.fsx

```diff
@@ -0,0 +1,30 @@
+open System.IO
+
+let EolAtEof(path: FileInfo) = 
```

Comment:
@realmarv it's not a string anymore so param should be not called path

Reply:
Done